VariationalBayes/ExponentialFamilies.py

```diff
@@ -116,6 +117,37 @@ def get_e_log_dirichlet(alpha):
     digamma_sum = sp.special.digamma(np.sum(alpha, 0, keepdims=True))
     return sp.special.digamma(alpha) - digamma_sum
 
+gh_loc, gh_weights = hermgauss(4)
```

Comment:
Please don't set global variables like this, especially with a default that may not be correct for a particular application.

If you really want to set a default, make the number of GH points an argument to the function and then evaluate gh_loc and gh_weights in the function.  Also, please then add a comment with how you decided four was enough.